VideoLocker/src/main/java/org/edx/mobile/util/images/TopAnchorWidthFillTransformation.java

```diff
@@ -0,0 +1,49 @@
+package org.edx.mobile.util.images;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.os.Build;
+
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import com.bumptech.glide.load.resource.bitmap.BitmapTransformation;
+
+/**
+ * Created by miankhalid on 9/9/15.
+ */
```

Comment:
The point is that in a collaborative project, there is no one obvious/static author of a class, and the only way to keep track of the actual author of a particular code is via SCM history. Android Studio also shows a warning when using the default template without any editing.
> On 10-Sep-2015, at 3:01 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/src/main/java/org/edx/mobile/util/images/TopAnchorWidthFillTransformation.java <https://github.com/edx/edx-app-android/pull/354#discussion_r39143186>:
> 
> > @@ -0,0 +1,49 @@
> > +package org.edx.mobile.util.images;
> > +
> > +import android.content.Context;
> > +import android.graphics.Bitmap;
> > +import android.graphics.Canvas;
> > +import android.graphics.Paint;
> > +import android.os.Build;
> > +
> > +import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
> > +import com.bumptech.glide.load.resource.bitmap.BitmapTransformation;
> > +
> > +/**
> > + * Created by miankhalid on 9/9/15.
> > + */
> Seems fine to me, saves us from some clicks to know who is the culprit 
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/354/files#r39143186>.
>